cypress/repositories.bzl

```diff
@@ -33,10 +33,18 @@ load("@aspect_rules_cypress//cypress:toolchain.bzl", "cypress_toolchain")
 
 filegroup(
     name = "files",
-    srcs = select({
-        "@bazel_tools//src/conditions:darwin": ["Cypress.app"],
-        "//conditions:default": ["Cypress"],
-    }) + ["binary_state.json"],
+    # TODO(mrmeku): For some reason, using a directory is incompatible with RBE
```

Comment:
source directories are not compatible with RBE; you have make a copy of the directory to the output tree for Bazel to be able to send that to a remote executor

Reply:
Thanks for the info!

Reply:
I'm confused why the old rules_nodejs varaint of the cypress toolchain was able to work on RBE without copying logic, but I'm sure I'm just missing something